_posts/2018-01-21-site-reactions-and-webirc.md

```diff
@@ -24,6 +24,6 @@ The [`migrate` PR](https://github.com/ircv3/ircv3-specifications/pull/330) allow
 
 The [`message-status` PR](https://github.com/ircv3/ircv3-specifications/pull/319) lets clients indicate that they're typing, and whether a message has been delivered/read.
 
-The [`rename` PR](https://github.com/ircv3/ircv3-specifications/pull/308) lets users rename channels. This is especially useful on networks with stricter naming guidelines like Freenode.
```

Comment:
I don't think old posts should be changed. Libera didn't even exist in 2018, and it will mess up RSS readers

Reply:
RSS readers won't be messed up because of atom guids but yeah no need to change this particular line. The other post that links to the channel can be changed though.

Reply:
Reverted,